website/docs/setup/repos.md

```diff
@@ -159,3 +159,95 @@ need to use the following flag with the initial Corso `repo init` command:
 Corso also supports the use of object storage systems with no TLS certificate or with self-signed
 TLS certificates with the `--disable-tls` or `--disable-tls-verification` flags.
 [These flags](../../cli/corso-repo-init-s3) should only be used for testing.
+
+## Filesystem Storage
+
+Corso supports creating a repository on local or network attached filesystem.
+
+:::note
+Filesystem repositories aren't S3 compatible.
+:::
+
+### Filesystem storage prerequisites
+
+None.
+
+### Initialize filesystem repository
+
+Before first use, you need to initialize a Corso repository with `corso repo init filesystem`. See the command details
+[here](../../cli/corso-repo-init-filesystem). Corso will create the directory structure if necessary, including any
+missing parent directories.
+
+<Tabs groupId="os">
+<TabItem value="win" label="Powershell">
+
+  ```powershell
+  # Initialize the Corso Repository
+  $Env:CORSO_PASSPHRASE = 'CHANGE-ME-THIS-IS-INSECURE'
+  .\corso repo init filesystem --path C:\Users\user\corso-test
+  ```
+
+</TabItem>
+<TabItem value="unix" label="Linux/macOS">
+
+  ```bash
+  # Initialize the Corso Repository
+  export CORSO_PASSPHRASE="CHANGE-ME-THIS-IS-INSECURE"
+  ./corso repo init filesystem --path $HOME/corso-test
+  ```
+
+</TabItem>
+<TabItem value="docker" label="Docker">
+
+<CodeBlock language="bash">{
+`# Initialize the Corso Repository
+export CORSO_PASSPHRASE="CHANGE-ME-THIS-IS-INSECURE"
+docker run --env-file $HOME/.corso/corso.env \\
+  --volume $HOME/.corso:/app/corso ghcr.io/alcionai/corso:${Version()} \\
+  repo init filesystem --path /path/in/container`
+}</CodeBlock>
```

Comment:
I think this example needs work. 

With the above you will most likely endup with a local repo on the container file system which is going to be ephemeral. I think this is easy to miss for most people. 

I think you want to develop the example so that the location where a filesystem location from the host is mapped to a specific location inside the container. The simplest would be to map another volume

Reply:
Thanks! I've fixed this by adding a volume. Confirmed that this works. 

```
# Initialize the Corso Repository
export CORSO_PASSPHRASE="CHANGE-ME-THIS-IS-INSECURE"
docker run --env-file $HOME/.corso/corso.env \
  --volume $HOME/.corso:/app/corso ghcr.io/alcionai/corso:unreleased-280ba3ace \
  --volume /path/on/host/corso-repo:/path/in/container/corso-repo \
  repo init filesystem --path /path/in/container/corso-repo
```